src/main/java/com/fasterxml/jackson/databind/deser/impl/PropertyValueBuffer.java

```diff
@@ -176,6 +179,34 @@ public Object[] getParameters(SettableBeanProperty[] props)
                 }
             }
         }
+
+        // [databind#562]: Respect @JsonAnySetter in @JsonCreator
+        // check if we have anySetter Param
+        AnnotationIntrospector ai = _context.getAnnotationIntrospector();
```

Comment:
Alas, this is not acceptable from performance perspective: all annotation introspection must occur when constructing deserializers and related handlers. It should be possible to change handling to keep track of information about existence of "any setter", instead of trying to introspect for every deserialization call.